pkg/framework/resource/resource.go

```diff
@@ -93,3 +96,51 @@ func (r *Resource) InstallResource(name, values, channel string, conditions ...f
 
 	return nil
 }
+
+func (r *Resource) WaitForStatus(gsHelmClient *helmclient.Client, release string, status string) error {
+	operation := func() error {
+		rc, err := r.helmClient.GetReleaseContent(release)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+		if rc.Status != status {
+			return microerror.Maskf(releaseStatusNotMatchingError, "waiting for %q, current %q", status, rc.Status)
+		}
+		return nil
+	}
+
+	notify := func(err error, t time.Duration) {
+		log.Printf("getting release status %s: %v", t, err)
```

Comment:
Do we use this different logging for better readability? I always find it hard to tell where logs are coming from.

Reply:
Lets please use the configured logger and the usual log structure.